models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -103,7 +107,55 @@ kyberswap_dex AS (
         ,evt_tx_hash                                                       AS tx_hash
         ,''                                                                AS trace_address
         ,evt_index
-    FROM {{ source('kyber_avalanche_c', 'MetaAggregationRouter_evt_Swapped') }}
+    FROM {{ source('kyber_avalanche_c_trades', 'AggregationRouterV2_evt_Swapped') }}
+    WHERE
+        {% if is_incremental() %}
+        evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% else %}
+        evt_block_time >= '{{ project_start_date }}'
+        {% endif %}
+
+    UNION ALL
+
+    SELECT
+        'v3'                                                               AS version
+        ,evt_block_time                                                    AS block_time
+        ,sender                                                            AS taker
+        ,''                                                                AS maker
+        ,returnAmount                                                      AS token_bought_amount_raw
+        ,spentAmount                                                       AS token_sold_amount_raw
+        ,cast(NULL as double)                                              AS amount_usd
+        ,dstToken                                                          AS token_bought_address
+        ,srcToken                                                          AS token_sold_address
+        ,contract_address                                                  AS project_contract_address
+        ,evt_tx_hash                                                       AS tx_hash
+        ,''                                                                AS trace_address
+        ,evt_index
+    FROM {{ source('kyber_avalanche_c_trades', 'AggregationRouterV3_evt_Swapped') }}
```

Comment:
We may need to remove `AggregationRouterV3` and `AggregationRouterV2` as currently these tables are unavailable on dune V2 avalanche_c chain. Maybe we can submit the contract for decoding.